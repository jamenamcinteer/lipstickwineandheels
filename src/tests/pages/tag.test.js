import React from 'react'
import { shallow } from 'enzyme'
import TagIndex from '../../pages/tag'
import data from '../fixtures/data'

test('should render tag page correctly', () => {
  const wrapper = shallow(<TagIndex data={data.data} />)
  expect(wrapper).toMatchSnapshot()
})
