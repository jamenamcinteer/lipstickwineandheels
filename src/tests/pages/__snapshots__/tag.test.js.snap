// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render tag page correctly 1`] = `
<PageTransition>
  <div
    style={
      Object {
        "background": "#fff",
      }
    }
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
    >
      <title>
        Topics | 
        Lipstick, Wine, and Heels
      </title>
      <html
        lang="en"
      />
    </HelmetWrapper>
    <div
      className="heroContainer"
    >
      <div
        className="hero"
      >
        <h1>
          Topics
        </h1>
      </div>
    </div>
    <div
      className="wrapper"
    >
      <ul
        className="topic-list"
      >
        <li
          key="tech"
        >
          <div
            className="imageContainer"
          />
          <div
            className="contentContainer"
          >
            <h2
              className="previewTitle"
            >
              <GatsbyLink
                to="/tag/tech"
              >
                Tech
              </GatsbyLink>
            </h2>
          </div>
        </li>
        <li
          key="travel"
        >
          <div
            className="imageContainer"
          />
          <div
            className="contentContainer"
          >
            <h2
              className="previewTitle"
            >
              <GatsbyLink
                to="/tag/travel"
              >
                Travel
              </GatsbyLink>
            </h2>
          </div>
        </li>
      </ul>
    </div>
  </div>
</PageTransition>
`;
